client/disputes/test/__snapshots__/index.tsx.snap

```diff
@@ -460,10 +460,8 @@ exports[`Disputes list renders correctly 1`] = `
                   class="woocommerce-table__item is-left-aligned"
                 >
                   <a
-                    class="woocommerce-table__clickable-cell"
                     data-link-type="wc-admin"
-                    href="admin.php?page=wc-admin&path=%2Fpayments%2Fdisputes%2Fdetails&id=dp_asdfghjkl"
-                    tabindex="-1"
+                    href="https://shop.local"
```

Comment:
Nitpick: It may help to have tests to check both scenarios - `customer_url` and no `customer_url`.

Reply:
@anu-rock I was thinking about this when working on the tests and found two solutions, but couldn't decide what's the best given I'm new to the code-base:

1. Add a test case here, without the `customer_url`. But this assumes any future changes will be added here, which will make the snapshot a little bigger.
2. Maybe expose a smaller component? Like DisputeRow or just Dispute in addition to the currently exposed DisputesList

Both solutions will work. The first is easier for now, but the second one is probably more useful for future changes 🤔 

Anyway, happy to go forward (part of this PR or a new PR) with whichever is the best option

Reply:
@elazzabi Glad to know you considered all possibilities here 👍 

> Add a test case here, without the customer_url. But this assumes any future changes will be added here, which will make the snapshot a little bigger.

You are right in assuming that more changes will likely happen. Props will be added and removed in time. That said, I do not expect a lot of conditional props (like `customer_url`) to be added. In that sense, it may not hurt to fully cover this prop.

> Maybe expose a smaller component? Like DisputeRow or just Dispute in addition to the currently exposed DisputesList

Good thought. At this point, it may be a bit overkill.

Ultimately, it's just a nitpick and a pretty minor one. I'll leave the decision to you :) It's okay to skip it for now.

Reply:
That sounds good! I pushed a fix here to add a new test case https://github.com/Automattic/woocommerce-payments/pull/3723/commits/10ee24e5c4ce63795d69ff9a7a417a8eb9b858d3